pkg/alertmanager/multitenant_test.go

```diff
@@ -2555,39 +2565,38 @@ func TestComputeConfig(t *testing.T) {
 			name: "no grafana configuration, custom mimir config",
 			cfg: alertspb.AlertConfigDescs{
 				Mimir: alertspb.AlertConfigDesc{
-					User:      "user-grafana",
```

Comment:
I'm just replacing the `"user-grafana"` string for the `testTenant` variable.
No relevant changes until line 3427.